.github/workflows/release.yml

```diff
@@ -0,0 +1,80 @@
+name: "Experimental Release"
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Push tag
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v5.5
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          custom_tag: ${{ github.run_number }}
+          tag_prefix: experimental-
+      - name: Create release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.tag_version.outputs.new_tag }}
+          release_name: "Experimental Build #${{ github.run_number }}"
+          body: |
+            These are the outputs for the experimental build of commit [${{ github.sha }}](https://github.com/${{ github.repository }}/commit/${{ github.sha }})
+          draft: false
+          prerelease: true
+  windows:
+    needs: release
+    strategy:
+      matrix:
+        include:
+          - name: Windows Tiles x64
+            mxe: x86_64
+            artifact: windows-tiles-x64
+          - name: Windows Tiles x32
+            mxe: i686
+            artifact: windows-tiles-x32
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          sudo apt install astyle autoconf automake autopoint bash bison bzip2 cmake flex gettext git g++ gperf intltool libffi-dev libgdk-pixbuf2.0-dev libtool libltdl-dev libssl-dev libxml-parser-perl lzip make mingw-w64 openssl p7zip-full patch perl pkg-config python ruby scons sed unzip wget xz-utils g++-multilib libc6-dev-i386 libtool-bin
+      - name: Install MXE
+        run: |
+          sudo apt install software-properties-common lsb-release
+          sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 86B72ED9
+          sudo add-apt-repository "deb [arch=amd64] https://pkg.mxe.cc/repos/apt `lsb_release -sc` main"
+          sudo apt update
+          sudo apt install mxe-{i686,x86-64}-w64-mingw32.static-{sdl2,sdl2-ttf,sdl2-image,sdl2-mixer,gettext}
+      - name: Build CDDA
+        env:
+          PLATFORM: /usr/lib/mxe/usr/bin/${{ matrix.mxe }}-w64-mingw32.static-
+        run: |
+          cat >VERSION.txt <<EOL
+          build type: ${{ matrix.artifact }}
+          build number: ${{ github.run_number }}
+          commit sha: ${{ github.sha }}
+          commit url: https://github.com/${{ github.repository }}/commit/${{ github.sha }}
+          EOL
+
+          make -j$((`nproc`+0)) CROSS="${PLATFORM}" TILES=1 SOUND=1 RELEASE=1 LOCALIZE=1 BACKTRACE=0 PCH=0 bindist
```

Comment:
`lang` folder is missing in the output binary compared to Jenkins builds. You probably need to add `LANGUAGES=all` here

Reply:
Ah nice -- I have that change testing [here](https://github.com/reasonablytall/Cataclysm-DDA/actions/runs/813316453). Once it looks good I'll make open a new PR.